pkgdb2client/admin.py

```diff
@@ -209,10 +217,13 @@ def do_list(args):
     data = PKGDBCLIENT.handle_api_call('/admin/actions/', params=data)
 
     cnt = 0
+    ids = []
     for cnt, action in enumerate(data['actions']):
         print _action2msg(action)
+        ids.append(action["id"])
 
     print 'Total: {0} actions'.format(cnt + 1)
```

Comment:
This seems.. untrue.  If zero actions are listed here, then it will print '1' action.